recipe/meta.yaml

```diff
@@ -14,13 +14,11 @@ source:
     sha256: a9598abd5c697e4a49a512b285c5747413d1f45155a12ecb1f5ee389c0459afa
     folder: ucx
     patches:
-      - patches/PR_4137.patch
-      - patches/PR_4366.patch
       - patches/PR_4383.patch
       - patches/PR_4384.patch
```

Comment:
These were already in `v1.7.x` before PR ( https://github.com/openucx/ucx/pull/4515 ) was made. So we are hanging on to them until the next RC.